sw/host/ot_certs/tests/generic.hjson

```diff
@@ -112,6 +115,9 @@
         },
         authority_key_identifier: { var: "auth_key_id" },
         subject_key_identifier: { var: "pub_key_id" },
+        basic_constraints: {
+            ca: { var: "basic_constraints_ca" }
+        }
         extensions: [
```

Comment:
lines 118..120 how is this supposed to work if the certificate does not have to include the basic constraints extension?

Maybe adding a test case showing it would help.

Reply:
Sure, you just omit the `basic_constraints` block entirely, since it's an `Option` in the main cert.